app/actions/service_offering_delete.rb

```diff
@@ -3,7 +3,7 @@ class ServiceOfferingDelete
     class AssociationNotEmptyError < StandardError; end
 
     def delete(service_offering_model)
-      association_not_empty! unless service_offering_model.service_plans.empty?
+      association_not_empty! unless ServicePlan.where(service_id: service_offering_model.id).empty?
```

Comment:
`if ... any?` instead of `unless empty?`

Reply:
Sure